olp-cpp-sdk-dataservice-read/tests/SerializerTest.cpp

```diff
@@ -426,4 +429,84 @@ TEST(SerializerTest, VersionResponse) {
   ASSERT_EQ(expectedOutput, json);
 }
 
+TEST(SerializerTest, ConsumerProperties) {
```

Comment:
It's not clear what is this test about. Is it to test the `ConsumerProperties` constructor or the `olp::serializer::serialize` method?

Consider splitting it into single-assertion tests or using sub-scopes to cut the test in parts.

Reply:
I removed the corresponding test about testing `CustomProperties` methods. I think we need to implement them in separate ticket/PR, because this PR about serializing.